package.json

```diff
@@ -40,15 +39,18 @@
     "optionalOkapiInterfaces": {
       "actual-cost-record-storage": "0.4",
       "automated-patron-blocks": "0.1",
+      "batch-print": "1.0",
       "circulation": "9.0 10.0 11.0 12.0 13.0 14.0",
       "consortia": "1.0",
       "feesfines": "16.1 17.0 18.0 19.0",
       "loan-policy-storage": "1.0 2.0",
       "loan-storage": "4.0 5.0 6.0 7.0",
+      "add-info": "0.1",
```

Comment:
petty: please preserve the alphabetization here. I recognize this is not your fault since the original branch was unsorted, but somebody sorted our branch, and we might as well keep it. The alphabet is so nice.